packages/playwright-core/src/server/firefox/ffNetworkManager.ts

```diff
@@ -149,7 +149,7 @@ export class FFNetworkManager {
   }
 }
 
-const causeToResourceType: {[key: string]: string} = {
+const causeToResourceType: { [key: string]: string } = {
```

Comment:
Oops! Will remove these from final merge. I had an errant auto-format enabled on my editor, and it coincidentally differs but does not violate eslint rules.

Reply:
If you find a eslint rule for that which aligns it, we're more than happy to add it. Because currently there is none configured and vscode formatting != whats used in the code base and eslint accepts both.